### ISSUE TITLE
2 of 3 RPC endpoints are failing

### ISSUE THREAD

**Issue by modenero** [SoftwareVerde/pos-cash#1]
tl:dr as it stands right now, after 60 seconds, the merchant has no way of being notified of payment receipt.

There are 3 RPC endpoints being utilized for on-chain tx activity in this application:
- `listenForTransactionsViaBitcoinVerdeDotOrg`
- `listenForTransactionsViaBitcoinDotCom`
- `listenForTransactionsViaBlockchainDotInfo`

Of the 3, only Blockchain.Info is working correctly for the purpose of payment notification.

Unfortunately, the websocket connection to Blockchain.Info seems to automatically timeout after 60 seconds. This leaves no ability to provide notification of payment receipt to the merchant.

**edit:**

this BitcoinVerde explorer is currently timing-out:
https://explorer.bitcoinverde.org/api/v1/transactions/6D25D725402BD60C12C86E295DE4DB9AB41035C981A5421F8F5C9EA43F318C2A

```
{"errorMessage":"Request timed out.","wasSuccess":0}
```

this is likely the cause of the problem with the `BitcoinVerdeDotOrg` endpoint.

**Comment by joshmg** [comment]
I'm a bit late to the party, but I'll investigate this.

**Comment by joshmg** [comment]
Okay, so there's nothing I can do to bring back Bitcoin.com's api--but IIRC, Troutner has a similar service we could look into expanding, but I'm leaving that for another day.

The bitcoinverde.org websocket timeouts are a server-side issue that is currently being resolved.  I'll revisit this in a few days to confirm it's actually resolved.

I've pushed commit 5c62b6b; with it, the blockchain.info websocket will now reconnect as long as it's been alive for 5 seconds (to prevent spamming them if they ever discontinue their service).

**Comment by modenero** [comment]
oh fantastic!

i actually much prefer this "web" version to the Bitcoin Cash Register. i just don't believe the "native" version adds any additional value, so it's very much appreciated to have a (browser-based) option once again..

cheers!
